components/admin_console/billing/billing_subscriptions.scss

```
@@ -196,3 +196,57 @@
         text-decoration: none;
     }
 }
+
+.customerCancelSubscriptionSection {
```

nit: I would drop the word `customer` on all these names so they are not so wordy

```suggestion
.cancelSubscriptionSection {
```

---

agree. Done!